apps/native/app/src/ui/lib/link/link-text.tsx

```diff
@@ -1,24 +1,48 @@
 import React from 'react'
 import styled, { useTheme } from 'styled-components/native'
-import { Typography } from '../typography/typography'
+import { ImageSourcePropType, Text, TextStyle } from 'react-native'
+import { fontByWeight } from '@ui/utils'
 
 const Host = styled.View`
   border-bottom-width: 1px;
   border-bottom-color: ${(props) => props.theme.color.blue400};
   margin-bottom: 1px;
 `
 
+const variants = {
+  small: { fontSize: 12, lineHeight: 16, fontWeight: '600' },
+  large: { fontSize: 16, lineHeight: 20, fontWeight: '600' },
+}
+
+type LinkVariant = keyof typeof variants
+
 interface LinkTextProps {
   children: string | React.ReactNode
+  variant?: LinkVariant
+  icon?: React.ReactNode | ImageSourcePropType
 }
 
-export function LinkText({ children }: LinkTextProps) {
+export function LinkText({ children, variant = 'large' }: LinkTextProps) {
   const theme = useTheme()
+  const styles = (variants as Record<string, TextStyle>)[variant]
+  const fontSize = styles.fontSize ?? 15
```

Comment:
fontSize should always be defined. Where does this 15 come from

Reply:
sorry, copy paste from Typography component 😄